modules/openapi-generator/src/main/resources/typescript-axios/apiInner.mustache

```diff
@@ -189,7 +189,7 @@ export const {{classname}}AxiosParamCreator = function (configuration?: Configur
             localVarUrlObj.query = {...localVarUrlObj.query, ...localVarQueryParameter, ...options.query};
             // fix override query string Detail: https://stackoverflow.com/a/7517673/1077943
             delete localVarUrlObj.search;
-            localVarRequestOptions.headers = {...localVarHeaderParameter, ...options.headers};
+            localVarRequestOptions.headers = {...localVarHeaderParameter, ...baseOptions.headers, ...options.headers};
```

Comment:
- Based on the code above, it seems that `baseOptions` can be undefined - probably need null check here before accessing `baseOptions.headers`.
- Shouldn't `localVarHeaderParameter` have priority over `baseOptions.headers`? Right now it looks like base options can override customizations specific to a particular api method

Reply:
1) I will fix this
2) I dont know, but I think localVarHeaderParameter will override baseOptions, and you can't change variables in localVarHeaderParameter in no way. So my option is allow override localVarHeaderParameter by baseOptions setted by user for all, and they overriding by options from request.